docker/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG target
 
-FROM $target/node:12-alpine
+FROM $target/node:12-alpine3.11
```

Comment:
Just one last question, why can't we use the latest node:12-alpine image? (just asking)

Reply:
Looking at the commits history it was to fix the OZW version without using Git. But then, we decided to move on manual compilation on a given commit.

So this constraint is not needed anymore.

I'll remove it.

Reply:
Done